lib/cartopy/io/ogc_clients.py

```diff
@@ -78,20 +79,115 @@
     'urn:ogc:def:crs:OGC:1.3:CRS84': _WGS84_METERS_PER_UNIT,
 }
 
-_URN_TO_CRS = {
-    'urn:ogc:def:crs:EPSG::27700': ccrs.OSGB(),
+_URN_TO_CRS = collections.OrderedDict({
```

Comment:
I don't think initializing an `OrderedDict` from an _unordered_ `dict` will preserve the order you intend.